src/prx/main.py

```diff
@@ -253,41 +253,43 @@ def format_flat_rows(row):
     #  - time-of-flight: around 70 ms for MEO orbits. This includes small
     #  terms (up to tens of meters in units of distance, ten meters correspond to 34 nanoseconds)
     #  such as satellite code bias and atmospheric delays
-    #  - receiver clock offset w.r.t. the satellite's constellation time (GPST, GST, BDT etc.)
     #  - satellite clock offset w.r.t. its constellation time
-    # By subtracting it from the receiver time of reception, we get the time of emission in
-    # the satellite's constellation time frame, plus the satellite system time clock offset plus those small terms
-    # of a few tens of nanoseconds.
+    #  - the receiver clock offset w.r.t. the satellite's constellation time (GPST, GST, BDT etc.) modulo 1 second
+    # The integer seconds of the receiver clock offset w.r.t. the satellite's constellation time (GPST, GST, BDT etc.)
+    # are likely removed by the receiver to align all pseudoranges to the same order of magnitude.
+    # By subtracting the term from the receiver time of reception, we get the time of emission in
+    # the satellite's constellation time frame (integer-second aligned to the receiver time frame), plus the
+    # satellite clock offset plus those small terms of a few tens of nanoseconds
     tof_dtrx = pd.to_timedelta(
         per_sat[code_phase_columns]
         .mean(axis=1, skipna=True)
         .divide(constants.cGpsSpeedOfLight_mps),
         unit="s",
     )
-    per_sat["time_of_emission_in_satellite_time"] = (
-        per_sat["time_of_reception_in_receiver_time"]
-        - per_sat.system_time_scale_epoch
-        - tof_dtrx
+    per_sat[
+        "time_of_emission_in_satellite_time_integer_second_aligned_to_receiver_time"
+    ] = per_sat["time_of_reception_in_receiver_time"] - tof_dtrx
+    # As error terms are tens of nanoseconds here, and the receiver clock is integer-second aligned to GPST, we
+    # already have times-of-emission that are integer-second aligned GPST here.
+    per_sat["time_of_emission_isagpst"] = (
+        per_sat.time_of_emission_in_satellite_time_integer_second_aligned_to_receiver_time
     )
-    per_sat["time_of_emission_weeksecond_system_time"] = per_sat.apply(
+    per_sat["time_of_emission_weeksecond_isagpst"] = per_sat.apply(
```

Comment:
I would suggest to rename this parameter `time_of_emission_in_seconds_of_week_isagpst`.

Reply:
See https://github.com/jtec/prx/pull/70#discussion_r1554713805